index.bs

```diff
@@ -2710,23 +2676,19 @@ The steps for [=issuing a credential request to an authenticator=] are as follow
                             Then, using |transport|, invoke the [=authenticatorGetAssertion=] operation on
                             |authenticator|, with |rpId|, |clientDataHash|, |allowCredentialDescriptorList|,
                             |userVerification|,
-                            |enterpriseAttestationPossible|,
-                            |attestationFormats|,
                             and |authenticatorExtensions| as parameters.
 
                         :   [=list/is empty=]
                         ::  Using local configuration knowledge of the appropriate transport to use with |authenticator|,
                             invoke the [=authenticatorGetAssertion=] operation on |authenticator| with |rpId|,
                             |clientDataHash|, |allowCredentialDescriptorList|, |userVerification|,
-                            |enterpriseAttestationPossible|, |attestationFormats|, and |authenticatorExtensions| as parameters.
+                            and |authenticatorExtensions| as parameters.
                     </dl>
 
             :   [=list/is empty=]
             ::  Using local configuration knowledge of the appropriate transport to use with |authenticator|, invoke the
                 [=authenticatorGetAssertion=] operation on |authenticator| with |rpId|, |clientDataHash|,
                 |userVerification|,
-                |enterpriseAttestationPossible|,
-                |attestationFormats|,
```

Comment:
These need to be deleted from the [authenticatorGetAssertion parameters](https://github.com/w3c/webauthn/pull/1997/files#diff-5e793325cd2bfc452e268a4aa2f02b4024dd9584bd1db3c2595f61f1ecf7b985R4736-R4739) too.

Reply:
Yep, and more needs to be tidied up from there too. Done.